pkg/agent/run.go

```diff
@@ -44,6 +44,13 @@ func Run(cmd *cobra.Command, args []string) {
 		log.Fatalf("Failed to parse config file: %s", err)
 	}
 
+	// AuthToken flag takes preference over token in configuration file.
+	if AuthToken == "" {
+		AuthToken = config.Token
+	} else {
+		log.Printf("Using authorization token from flag.")
```

Comment:
Is the issue here that it'll print both `Using authorization`  then potentially `A credentials file was specified. Using OAuth2 authentication` later?

Reply:
I guess this is a minor issue as authtoken will be gone soon.

Reply:
If someone uses both the token and the credentials file, it is going to use the credentials file and yes, print both messages.

It is not ideal but I didn't want to fail hard because I am often reusing the same config file with both auth methods.

I agree it might be confusing but this is going away soon.